sagan-site/src/main/java/sagan/projects/support/ProjectMetadataController.java

```diff
@@ -33,4 +40,87 @@ public Project projectMetadata(@PathVariable("projectId") String projectId) thro
         return service.getProject(projectId);
     }
 
+    @RequestMapping(value = "/{projectId}/**", method = GET)
```

Comment:
This is just the way the existing endpoint was defined. I didn't want to change a convention (I would prefer `@GetMapping` anyway).